### ISSUE TITLE
"./configure --enable-debug=yes" causes make failure

### ISSUE THREAD

**Issue by p-alik** [gearman/gearmand#42]
- Ubuntu 14.04
```bash
...
/usr/include/x86_64-linux-gnu/gnu/stubs-64.h
/usr/include/x86_64-linux-gnu/gnu/stubs.h
make[2]: *** [libhashkit/libhashkit_libhashkitinc_la-strerror.lo] Error 1
make[2]: Leaving directory `/vagrant/gearmand'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/vagrant/gearmand'
make: *** [all] Error 2
```
- Ubuntu 16.04
```bash
...
/usr/include/x86_64-linux-gnu/gnu/stubs-64.h
/usr/include/x86_64-linux-gnu/gnu/stubs.h
Makefile:6207: recipe for target 'libgearman/libgearman_libgearman_la-actions.lo' failed
make[2]: *** [libgearman/libgearman_libgearman_la-actions.lo] Error 1
make[2]: Leaving directory '/home/palik/workspace/Gearman/gearmand'
Makefile:8369: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/home/palik/workspace/Gearman/gearmand'
Makefile:3088: recipe for target 'all' failed
make: *** [all] Error 2
```

**Comment by chenzi2015** [comment]
- Centos 6.5

/usr/lib/gcc/x86_64-redhat-linux/4.4.7/../../../../include/c++/4.4.7/cerrno
/usr/lib/gcc/x86_64-redhat-linux/4.4.7/../../../../include/c++/4.4.7/clocale
/usr/lib/gcc/x86_64-redhat-linux/4.4.7/../../../../include/c++/4.4.7/cstdarg
/usr/lib/gcc/x86_64-redhat-linux/4.4.7/../../../../include/c++/4.4.7/cwctype
/usr/lib/gcc/x86_64-redhat-linux/4.4.7/../../../../include/c++/4.4.7/tr1_impl/cinttypes
/usr/lib/gcc/x86_64-redhat-linux/4.4.7/../../../../include/c++/4.4.7/tr1_impl/cstdint
/usr/lib/gcc/x86_64-redhat-linux/4.4.7/../../../../include/c++/4.4.7/x86_64-redhat-linux/bits/ctype_base.h
/usr/lib/gcc/x86_64-redhat-linux/4.4.7/../../../../include/c++/4.4.7/x86_64-redhat-linux/bits/ctype_inline.h
make[2]: *** [libtest/skiptest.o] 错误 1
make[2]: Leaving directory `/opt/soft/gearman-src/gearmand-1.1.14'
make[1]: *** [all-recursive] 错误 1
make[1]: Leaving directory `/opt/soft/gearman-src/gearmand-1.1.14'
make: *** [all] 错误 2

**Comment by p-alik** [comment]
resolved in #186
